resources/lib/data.py

```diff
@@ -151,6 +160,7 @@
         youtube=[
             dict(label='Studio Brussel', url='https://www.youtube.com/user/StuBru'),
         ],
+        epg_logo='https://images.vrt.be/orig/2019/03/12/1e383cf5-44a7-11e9-abcc-02b7b76bf47f.png',
```

Comment:
You can also use `height100`: https://images.vrt.be/height100/2019/03/12/1e383cf5-44a7-11e9-abcc-02b7b76bf47f.png

Reply:
Thanks! I just decided to go with higher quality logos. And while I can find **width160**, **width320**, **width640** and **width1280**, these are often incorrectly cropped. So using the **orig** logos looks best.